src/renderer/lib/cast.js

```diff
@@ -328,6 +316,18 @@ function dlnaPlayer (player) {
   }
 }
 
+function handleStatus (err, status) {
+  if (err || !status) {
+    return console.log('error getting %s status: %o',
```

Comment:
* We were logging before
* Looks like it's common for at least one of the cast libraries to call back with a null error **and** a null status object. I want to ignore when that happens
* Getting an error is also common. For example, if someone's using Chromecast, and then turns off the TV (powering off the Chromecast), then I think WebTorrent will get an error on the next status callback. That's expected.